### DISCUSSION TITLE
Hikvision

### DISCUSSION THREAD

**Discussion by cirrusflyer** [koush/scrypted#109] [Q&A] [answered]
Can someone with a Hikvision camera tell me how they enabled the onboard motion detection?  What I've done doesn't seem to work.

**Comment by mrMiimo** [comment, marked as answer]
you can do it from:
- NVR
- NVR management web page
- or use the iVMS-4200 app

Basically you need to enable the option and draw a grid of the area you want.
I am attaching a file of the iVMS and webpage.

![stepbystep-iVMS4200](https://user-images.githubusercontent.com/49750870/149126607-c5955875-42d9-4dff-9cda-3b8cd9add536.png)
![Schermata_2021-12-01_alle_20 38 03](https://user-images.githubusercontent.com/49750870/149126641-aaf4dffe-5eee-4b0d-8698-5d8820904ab6.png)
